source/common/ssl/context_impl.cc

```diff
@@ -131,6 +132,17 @@ ContextImpl::ContextImpl(ContextManagerImpl& parent, Stats::Scope& scope,
     verify_mode = SSL_VERIFY_PEER | SSL_VERIFY_FAIL_IF_NO_PEER_CERT;
   }
 
+  if (!config.verifyCertificateSpkiList().empty()) {
+    for (auto hash : config.verifyCertificateSpkiList()) {
+      const auto& decoded = Base64::decode(hash);
```

Comment:
reference to temporary again. I'm actually curious at this point if the compiler basically ignores the '&' in this case.

Reply:
Done.